src/desktop/apps/auction/__tests__/utils/renderTestComponent.js

```diff
@@ -44,12 +57,12 @@ export default function renderTestComponent ({ Component, data = {}, props = {},
 
   const wrapper = renderMode(
     <Provider store={store}>
-      <Component {...props} />
+      <Component {...props} />{' '}
```

Comment:
Can remove this the space (sometimes prettier injects this if it finds a `&nbsp;`